tests/external_processing/test_apm.py

```diff
@@ -0,0 +1,55 @@
+from utils import weblog, interfaces, scenarios, features
+
+from tests.test_config_consistency import *
```

Comment:
With that, it'll duplicate base test class. Let meet with zoom to see the best way to achieve this.

Reply:
```python
from tests import test_semantic_conventions as base_semantic_conventions


@scenarios.external_processing
class Test_Meta(base_semantic_conventions.Test_Meta):
    @irrelevant(True, reason="The meta component tag would always be http.request and not external-processing")
    def test_meta_component_tag(self):
         super().test_meta_component_tag()
    
@scenarios.external_processing 
class Test_EnvVar(base_semantic_conventions.Test_Meta):
    pass
```

Reply:
I rewrote all the tests to use duplicated base test classes 😄